MIGRATION.md

```diff
@@ -0,0 +1,37 @@
+# dp-frontend-renderer migration
+  
+====================
```

Comment:
As you have a '#' on line 1, we don't need this line anymore - `=======` is just an alternative to an h1/#